src/scheduled/weeklyStatistics.ts

```diff
@@ -1,199 +1,153 @@
-				(postsByNewMembers / threads.length) * 100
-			)}% of posts by new members`
-		);
+		const memberEmbed = getDefaultEmbed()
+			.setTitle('Posts from New Members')
+			.setDescription(
+				`${newMembers.size} new members posting in <#${supportChannelId}>\n` +
+					`${Math.round((postsByNewMembers / threads.length) * 100)}% of posts by new members`
+			);
 		embeds.push(memberEmbed);
 
 		let description = '';
 		let embedCount = 0;
-
-		let tags = Object.fromEntries(
-			Object.entries(unsortedTags)
-				.sort((a, b) => {
-					return unsortedTags[a[0]][a[0]] - unsortedTags[b[0]][b[0]];
-				})
-				.reverse()
+		const tags = Object.fromEntries(
+			Object.entries(unsortedTags).sort((a, b) => unsortedTags[b[0]][b[0]] - unsortedTags[a[0]][a[0]])
 		);
 
 		for (const tagId in tags) {
 			const tagName = await getTagName(guild, forum.availableTags, tagId);
 			let localDescription = `**${tagName}** (${tags[tagId][tagId]})\n`;
 
-			/** Sub tags sorted descending by count, excluding tags that show up just once. */
-			// const subTags = Object.entries(tags[tagId])
-			// 	.sort(([, countA], [, countB]) => countB - countA)
-			// 	.filter(([subTagId, count]) => subTagId !== tagId && count > 1);
-
-			// if (subTags.length) {
-			// 	const subDescriptions = [];
-			// 	for (const [id, count] of subTags) {
-			// 		const subTagName = await getTagName(guild, forum.availableTags, id);
-			// 		subDescriptions.push(`${subTagName} (${count})`);
-			// 	}
-			// 	localDescription += `+ ${subDescriptions.join(' / ')}\n`;
-			// }
-
 			localDescription += '\n';
 
 			if (description.length + localDescription.length > 4096) {
-				let embed = getDefaultEmbed();
-
-				if (embedCount == 0) {
-					embed.setTitle('Tags');
-				}
-
-				embed.setDescription(description);
-				description = '';
+				const embed = getDefaultEmbed()
+					.setTitle(embedCount === 0 ? 'Tags' : '')
+					.setDescription(description);
 				embeds.push(embed);
-				embedCount += 1;
+				description = '';
+				embedCount++;
 			}
 
 			description += localDescription;
 		}
 
-		let embed = getDefaultEmbed();
-
-		if (embedCount == 0) {
-			embed.setTitle('Tags');
-		}
-
-		embed.setDescription(description || 'failed');
-		embeds.push(embed);
+		const finalEmbed = getDefaultEmbed()
+			.setTitle(embedCount === 0 ? 'Tags' : '')
+			.setDescription(description || 'No tags available');
+		embeds.push(finalEmbed);
 
 		if (errors.length) {
-			const errorEmbed = getDefaultEmbed();
-			errorEmbed.setTitle('Errors');
-			errorEmbed.setDescription(errors.map(([err, id]) => `<#${id}>: ${err}`).join('\n'));
+			const errorEmbed = getDefaultEmbed()
+				.setTitle('Errors')
+				.setDescription(errors.map(([err, id]) => `<#${id}>: ${err}`).join('\n'));
 			embeds.push(errorEmbed);
 		}
 
-		for (let i = 0; i < embeds.length; i++) {
-			channel.send({ embeds: [embeds[i]] });
+		for (const embed of embeds) {
+			await channel.send({ embeds: [embed] });
 		}
```

Comment:
while we're refactoring: why do we do a loop here? there is already an embeds field that takes an array